content/preferences-settings/general.md

```diff
@@ -27,12 +27,16 @@ GUI controls and text DPI
 
 # CSS Theme Modifications
 
-In addition to selecting a pre-built theme you can also apply additional CSS customisations of your own to tweak the look-and-feel of darktable. A text box is provided for this purpose.
+In addition to selecting a pre-built theme you can also apply additional CSS customisations of your own to tweak the look-and-feel of darktable. darktable provides a textbox for this purpose in general tab of main preferences. No need to set import lines of darktable CSS themes. You just need to choose which default darktable theme you want to start from (in theme menu just above) and then just copy/paste tweaks you find or create your own.
 
-When you have finished entering your CSS tweaks, click on the 'save CSS and apply' button. This will save your CSS to a file (`$HOME/.config/darktable/user.css`) and immediately apply it to the current darktable session.
+When you have finished entering your CSS tweaks, click on the 'save CSS and apply' button. This will save your CSS to a file (`$HOME/.config/darktable/user.css` on Mac and Linux ; `C:\%LOCALAPPDATA%\darktable\user.css` on Windows) and immediately apply it to the current darktable session.
 
 If your changes cause any issues, you can uncheck the "modify selected theme with CSS tweaks below" check box. This will immediately restore the base theme but will leave your tweaks in the editor so that you can re-edit them and try again. Simply press "save CSS and apply" again when you are ready to retry. This will automatically re-check the "modify selected theme with CSS tweaks below" checkbox and apply the new CSS.
 
-If you have any issues with darktable please retry with this option unchecked to be certain that they were not caused by any of your alterations.
+*If you have any issues with darktable please retry with this option unchecked to be certain that they were not caused by any of your alterations.*
 
 If you want to understand how darktable themes are constructed you are advised to look at the file `$DARKTABLE/share/darktable/themes/darktable.css` for more information (where `$DARKTABLE` is darktable's installation directory). You can also use the Gtk Inspector tool to analyse the existing layout.
+
+Now darktable CSS use classes for main parts and then ids for specific ones, like it is recommended by W3C specifications. Main darktable classes are now specified in darktable.css header. Other ones could be found inside file content. To optimize CSS, set on text box only what you want to change. No need to copy parts already set in darktable.css file (or other theme you select). It's always better to set tweaks that will impact all items (for example: buttons) then set specific tweaks if you want to just set a precise tweak for specific button.
```

Comment:
I'm not 100% sure what you're trying to get across here...

- what do you mean by "main parts" and "specific ones"? I'm sure there's a better way of putting it but perhaps it needs to be more clear. No need to mention W3C specs IMO.
- what do you mean by "inside file content"?
- I'm not sure what you're trying to get across with "It's always better to set tweaks that will impact all items (for example: buttons) then set specific tweaks if you want to just set a precise tweak for specific button."

Reply:
I will update and simplify that part to follow your comments. Hope it will be better and clearer. For last point, It's just about CSS recommendations about what we never do before and I updated on CSS code. If you want to change the UI: for example, we set first CSS settings that will apply to all buttons. And after that only set some needed tweaks for specific buttons. So first CSS apply to a lot of items then set CSS settings that apply to only some. I hope you will find with that explanation the way to better write that in english.

Reply:
But you're talking about how the theme CSS works, rather than how a user should tweak it?

Reply:
it's the same. a user need to take care of that to tweak css